gettext-rs/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "gettext-rs"
+description = "GNU Gettext FFI binding for Rust"
+version = "0.5.0"
+authors = ["Konstantin Salikhov <koka58@yandex.ru>"]
```

Comment:
Feel free to include yourself to the authors already @Minoru :) You've done to this lib more than I did already!

Reply:
Thanks, will do :)